File: EncryptDecrypt.cpp
// ConsoleApplication3.cpp : Defines the entry point for the console application.
//

#include "stdafx.h"
#include "string.h"

char * EncryptDecrypt(char _input[]) {
	int count = strlen(_input);
	printf("%d \n", count);
	int i = 0;
	for (i = 0; i < count; i++) {
		_input[i] = ~_input[i];
	}
	_input[i] = '\0';

	return _input;
}

int main()
{
	int command = 0;
	char inputString[128] = { '\0' };
	char *tempString;
	while (true)
	{
		printf("Please Enter Command. 1(input)/2(result)/3(exit)\n");
		scanf_s("%d", &command);

		if (command == 1) {
			printf("Please Enter a String to encrypt.\n");
			scanf_s("%s", &inputString, 128);

			tempString = EncryptDecrypt(inputString);
			printf("Encrypted code is %s\n", tempString);
		}
		else if (command == 2) {
			EncryptDecrypt(inputString);
			printf("Decrypted code is %s\n", tempString);
		}
		else if (command == 3) {
			break;
		}
		else {
			continue;
		}

	}

	return 0;
}

File: EncryptDecrypt.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace StringEncrypt {
    class EncryptDecrypt {
        static string EnDe(string _input) {
            StringBuilder sb = new StringBuilder(_input);
            int i = 0;
            for(i = 0; i < sb.Length; i++) {
                sb[i] = (char)~sb[i];
            }

            return sb.ToString();
        }

        static void Main(string[] args) {
            
            int command = 0;
            string inputString = "";
            while(true) {
                Console.WriteLine("Please enter command. 0(Enter code)/1(Show En code)/2(Exit)");
                try {
                    command = Convert.ToInt32(Console.ReadLine());
                }
                catch(Exception e) {
                    continue;
                }

                Console.WriteLine("command: "+command.ToString());
                if(command == 0) {
                    Console.WriteLine("hello.");
                    inputString = Console.ReadLine();
                    inputString = EnDe(inputString);

                }
                else if(command == 1) {
                    Console.WriteLine("The encrypted string is "+ inputString.ToString());
                    Console.WriteLine("The origion string is " + EnDe(inputString));

                }
                else if(command == 2) {
                    break;
                }
                else {
                    continue;
                }


            }


        }
    }
}
